### ISSUE TITLE
Server container image builds referencing missing template

### ISSUE THREAD

**Issue by portante** [distributed-system-analysis/pbench#2990]
See https://github.com/distributed-system-analysis/pbench/blob/d4386635db9c9c3ef965d4175e627c3492261d95/server/containers/images/Makefile#L28

**Comment by webbnh** [comment]
Yes, I noticed that, too, some time ago.  However, I didn't worry about it because I expected that the CI work would shake it out, one way or another.  (And, we'll find out real soon now....)

I'm expecting this to be a non-issue, because, as I recall, the canned server container doesn't use this mechanism in its build.

Also, this is low priority, because I don't expect us to be shipping a containerized Pbench Server any time soon.  Moreover, when/if we do so, I expect that what we ship will be the base container for the canned server.  So, again, this issue won't have any impact.
